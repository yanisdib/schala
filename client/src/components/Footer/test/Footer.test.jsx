import { shallow } from 'enzyme';

import { Footer } from '../../';


describe('Footer component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<Footer />))

    it('renders correctly without crashing', () => expect(wrapper).toMatchSnapshot())
})