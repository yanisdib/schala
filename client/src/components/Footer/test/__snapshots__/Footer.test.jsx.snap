// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly without crashing 1`] = `
<footer>
  <div
    className="footer container"
  >
    <div
      className="footer-wrapper"
    >
      hello world!
    </div>
  </div>
</footer>
`;
